### ISSUE TITLE
autoimmune hepatitis

### ISSUE THREAD

**Issue by anastasia-coding** [obophenotype/human-phenotype-ontology#11344]
**Preferred term label:**
autoimmune hepatitis 

**Synonyms**
-----------

**Definition (free text, please give PubMed ID)**
Autoimmune hepatitis (AIH) is a chronic inflammatory disease of the liver characterised by elevated serum globulin levels and autoantibodies.

**Parent term (use hpo.jax.org/app)**
hepatitis

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Systemic lupus erythematosus (OMIM 152700)
Sjogren's syndrome (OMIM 270150)

**Your nano-attribution (ORCID)**
Anastasia-Vasiliki Madenidou (https://orcid.org/0000-0002-7382-9958)

**Comment by anastasia-coding** [comment]
PMID: 40276396

**Comment by pnrobinson** [comment]
@anastasia-coding this is more of a disease entity and should be coded as a Mondo or maybe an ORPHA code depending on rarity.
https://www.ncbi.nlm.nih.gov/books/NBK459186/
thus, this is outside of the score of the HPO
